src/javax/microedition/pim/FieldFullException.java

```diff
@@ -0,0 +1,39 @@
+/*
+	This file is part of FreeJ2ME.
+
+	FreeJ2ME is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	FreeJ2ME is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with FreeJ2ME.  If not, see http://www.gnu.org/licenses/
+*/
+package javax.microedition.pim;
+
+public class FieldFullException extends RuntimeException 
+{
+    
+	private final int fieldVal;
+
+	public FieldFullException() { fieldVal = -1; }
+
+	public FieldFullException(String detailMessage) 
+	{
+		super(detailMessage);
+		fieldVal = -1;
+	}
+
+	public FieldFullException(String detailMessage, int field) 
+	{
+		super(detailMessage);
+		this.fieldVal = field;
```

Comment:
Hmm... i've been stubbing those files for a few days so i don't remember exactly why things like those are in, but i named it "field" at the start and figured it would be better to just give it a unique name in the class, but probably didn't alter the rest of the code. I'll be removing those from all the files with unique fields i can find.